components/datadog/agentparams/params.go

```diff
@@ -187,6 +188,13 @@ func WithTelemetry() func(*Params) error {
 	}
 }
 
+func WithPulumiDependsOn(resources ...pulumi.ResourceOption) func(*Params) error {
```

Comment:
```suggestion
func WithPulumiResourceOptions(resources ...pulumi.ResourceOption) func(*Params) error {
```

Reply:
Fixed. Renamed the config member as well for consistency.